### PR TITLE
Lying down plays the bodyfall sfx

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
title
## Why It's Good For The Game
I think it's a nice element of sfx to the game
## Changelog
:cl: grungussuss
sound: lying down will now play the bodyfall sfx
/:cl:

### CODE REVIEW

**Comment by the-orange-cow** [comment]
I guess it could go either way, but intentionally lowing yourself onto the ground doesn't really seem like it should produce that sound.

**Comment by zxaber** [comment]
Maybe don't play it if the mob is sharing a tile with a bed?

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -724,6 +724,7 @@
 	if(layer == initial(layer)) //to avoid things like hiding larvas.
 		layer = LYING_MOB_LAYER //so mob lying always appear behind standing mobs
 	add_traits(list(TRAIT_UI_BLOCKED, TRAIT_PULL_BLOCKED, TRAIT_UNDENSE), LYING_DOWN_TRAIT)
+	playsound(src, SFX_BODYFALL, 50, TRUE)
```
Comment (MrMelbert) [inline code/modules/mob/living/living.dm:727]: This doesn't play the sound twice if someone gets shoved over, right?

**Review by MrMelbert — COMMENTED** [review]
I would say that willingly lying down should be quieter than being shoved over

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (MrMelbert) [reply code/modules/mob/living/living.dm:727]: Oh and does it get played if you buckle to something which requires lying down (beds)? Shouldn't if so

**Comment by HerCoyote23** [comment]
I don't like this because I feel that the bodyfall sfx currently carries an important message -- someone just got knocked over against their will. 
When I hear that sound, it gives me useful information about what just happened. I don't think we should sacrifice audio "readability" for the sake of realism/nice sfx.

**Comment by Sadboysuss** [comment]
good feedback, if I ever end up bringing this back I'll make a different sound for it

**Comment by MoonDeer728** [comment]
> good feedback, if I ever end up bringing this back I'll make a different sound for it

why you want add sound for this. its ok alredy

**Pull request closed without merging**